api/models/build.js

```diff
@@ -44,25 +44,22 @@ const beforeValidate = (build) => {
 
 const sanitizeCompleteJobErrorMessage = message => message.replace(/\/\/(.*)@github/g, '//[token_redacted]@github');
 
-const completeJobErrorMessage = (err) => {
-  let message = 'An unknown error occurred';
-  if (err) {
-    message = err.message || err;
-  }
-  return sanitizeCompleteJobErrorMessage(message);
-};
+const jobErrorMessage = ({ message = 'An unknown error occurred' }) => sanitizeCompleteJobErrorMessage(message);
```

Comment:
does this method need to take an object containing the `message` or can it just take the `message` as an argument?